### ISSUE TITLE
The component does not render if I pass a prop to data

### ISSUE THREAD

**Issue by escobar5** [gitim/react-native-sortable-list#53]
If I pass to data a static object it renders ok, but now I'm trying to pass a prop of my control, when I set data={this.props.content.options} then it shows nothing.

Do you have any idea what the problem might be?

this.props.content.options is an object like this:

```
{ 
  "-Ke6YjqvT4fVS-UThxkl" : { id = '-Ke6YjqvT4fVS-UThxkl', text = 'Sample text', value = 1 },
  ...
}
```

**Comment by gitim** [comment]
Maybe it is empty on initial pass and you add keys to it without rerender.

**Comment by escobar5** [comment]
No, it has values from the begining, can the problem be that it is outside the screen when first rendered? I'm using it inside a TabView, and that screen is not initially on screen

**Comment by i8wu** [comment]
Have you tried logging out the prop on mount? It might be that it's not actually being passed properly? If it it, are you overriding shouldComponentUpdate?

**Comment by escobar5** [comment]
Yes, also I'm using react-devtools and I see the items are there, just not visible:

<img width="1185" alt="screenshot 2017-06-15 14 12 16" src="https://user-images.githubusercontent.com/583737/27197676-c5e207be-51d4-11e7-98fb-c955d0e24b86.png">

**Comment by i8wu** [comment]
So the prop being passed is exactly the same as the static content you tested with? Are you using redux? If your reducers are impure the components may not actually update.

**Comment by escobar5** [comment]
I think I have better understanding of the issue now.

The problem is not with props or static content.

I have a TabView, with multiple tabs, there might be a lot of tabs, so I render only the previous, current and next, if the SortableList is in any of those initial tabs it renders, but if I have a SortableList in another tab that is not rendered at the beginning, then it's not rendered, no matter if I pass a prop or a static content. Right now I'm testing passing it a const with the row data.

**Comment by gitim** [comment]
could you test your app with changes from this branch https://github.com/gitim/react-native-sortable-list/tree/fix, maybe they will fix your issue

**Comment by gitim** [comment]
@escobar5 Could you test 0.0.15, hope it is fixing the bug.
